packages/client/modules/teamDashboard/components/UnpaidTeamModal/UnpaidTeamModal.tsx

```diff
@@ -1,40 +1,50 @@
-import {UnpaidTeamModal_viewer} from '../../../../__generated__/UnpaidTeamModal_viewer.graphql'
-import React from 'react'
 import styled from '@emotion/styled'
-import {createFragmentContainer} from 'react-relay'
 import graphql from 'babel-plugin-relay/macro'
-import {RouteComponentProps, withRouter} from 'react-router-dom'
+import React from 'react'
+import {createFragmentContainer} from 'react-relay'
 import DashModal from '../../../../components/Dashboard/DashModal'
 import DialogContent from '../../../../components/DialogContent'
 import DialogTitle from '../../../../components/DialogTitle'
 import IconLabel from '../../../../components/IconLabel'
 import PrimaryButton from '../../../../components/PrimaryButton'
-import withAtmosphere, {
-  WithAtmosphereProps
-} from '../../../../decorators/withAtmosphere/withAtmosphere'
+import useAtmosphere from '../../../../hooks/useAtmosphere'
+import useRouter from '../../../../hooks/useRouter'
+import {UnpaidTeamModal_viewer} from '../../../../__generated__/UnpaidTeamModal_viewer.graphql'
 
 const StyledButton = styled(PrimaryButton)({
   margin: '1.5rem auto 0'
 })
 
-interface Props extends WithAtmosphereProps, RouteComponentProps<{}> {
+interface Props {
   viewer: UnpaidTeamModal_viewer
 }
 
 const UnpaidTeamModal = (props: Props) => {
-  const {atmosphere, history, viewer} = props
+  const {viewer} = props
+  const atmosphere = useAtmosphere()
+  const {history} = useRouter()
   const {viewerId} = atmosphere
   const {team} = viewer
   if (!team) return null
-  const {teamName, organization} = team
-  const {orgId, billingLeaders, orgName} = organization
-  const billingLeaderName = billingLeaders[0].preferredName
+  const {name: teamName, organization, lockMessageHTML} = team
+  if (lockMessageHTML) {
```

Comment:
@jordanh i think this is what you were after?

Reply:
ah dope!